src/haddock/modules/topology/topoaa/defaults.yml

```
@@ -0,0 +1,3776 @@
+basic:
+  autohis:
+    default: true
+    type: bool
+    hover: No help yet
+    doc: No help yet
+intermediate:
+  delenph:
+    default: true
+    type: bool
+    hover: No help yet
+    doc: No help yet
+  randorien:
+    default: true
+    type: bool
+    hover: No help yet
+    doc: No help yet
+  log_level:
+    default: verbose
+    type: string
+    length: 9999
+    hover: No help yet
+    doc: No help yet
+  iniseed:
+    default: 917
+    type: integer
+    min: -9999
+    max: 9999
+    hover: No help yet
+    doc: No help yet
+  ligand_param_fname:
+    default: ''
+    type: path
+    length: 9999
+    hover: No help yet
+    doc: No help yet
+  ligand_top_fname:
+    default: ''
+    type: path
+    length: 9999
+    hover: No help yet
+    doc: No help yet
+guru:
+  limit:
+    default: true
+    type: bool
+    hover: No help yet
+    doc: No help yet
+  tolerance:
+    default: 0
+    type: integer
+    min: -9999
+    max: 9999
+    hover: No help yet
+    doc: No help yet
+  input:
+    mol1:
+      prot_segid:
```

input > mol1 or mol2 now looks like a dict of dicts while it is being parsed as an list of dicts. It would be nice to have a hint that it should be converted to a list.

Maybe something like
```suggestion
  input:
    type: list
    items:
      mol1:
        prot_segid:
```
or as mol1 and mol2 values are the same you could only define the type of items in the list once. This borrows from https://json-schema.org/understanding-json-schema/reference/array.html#items

```suggestion
  input:
    type: array
    index_prefix: mol
    items:
      prot_segid:
```